web/gallery/README.md

```diff
@@ -1,5 +1,32 @@
-A gallery of Flutter widgets and UX studies.
+# Flutter gallery
 
-Original source on [GitHub][gallery].
+A demo app for Flutter's material design and cupertino widgets, as
+well as many other features of the Flutter SDK.
 
-[gallery]: https://github.com/flutter/flutter/tree/master/examples/flutter_gallery
+## Building
+
+You can follow these instructions to build the gallery app
+and install it onto your device.
```

Comment:
nits: If we're going to include this purely as a web sample, this file needs some tweaks:

* "onto your device" isn't  accurate.
* build instructions should be for web.
* Android launcher icon reference at the bottom may be removable.